# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/reftests/webkit-gradient/reftest.list
# Commit: f733b37f9626
# Full Hash: f733b37f9626548ae83564a3fd43ddfa972977d9
# Author: Nicolas Silva <nsilva@mozilla.com>
# Date: 2021-04-22 21:31:57
# Regressor Bug: 1702228
# File Overlap Count: 1
# Description:
#   Bug 1702228 - Adjust reftests. r=gw
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D112970
# ==============================================================================

diff -r 0919633e25e9 -r f733b37f9626 layout/reftests/webkit-gradient/reftest.list
--- a/layout/reftests/webkit-gradient/reftest.list	Thu Apr 22 10:34:55 2021 +0000
+++ b/layout/reftests/webkit-gradient/reftest.list	Thu Apr 22 10:34:55 2021 +0000
@@ -10,7 +10,7 @@
 == webkit-gradient-linear-1b.html webkit-gradient-linear-1-ref.html
 == webkit-gradient-linear-1c.html webkit-gradient-linear-1-ref.html
 == webkit-gradient-linear-1d.html webkit-gradient-linear-1-ref.html
-== webkit-gradient-linear-2.html webkit-gradient-linear-2-ref.html
+fuzzy-if(webrender,0-1,0-150) == webkit-gradient-linear-2.html webkit-gradient-linear-2-ref.html
 
 # Tests for -webkit-gradient(radial, ...)
 == webkit-gradient-radial-1a.html webkit-gradient-radial-1-ref.html